.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "8"
+  - "10"
```

Comment:
Add 12?

Reply:
@davidlehn node 12 is complicated, from what I understand. (has a bunch of branches, upstream - like this one https://github.com/digitalbazaar/crypto-ld/pull/34 )

Reply:
Ok. I thought some of that was getting resolved. Could make a branch with just that change and poke travis periodically to see when it passes.

Reply:
Np, I'll give that a try.

Reply:
The crypto-ld node-12 branch is passing tests. Once published, at least that package won't be an issue.

Reply:
Added support for Node 12 to travis.